lib/backup/actions/copy.go

```diff
@@ -84,6 +84,7 @@ func runCopy(src common.OriginFS, dst common.RemoteFS, concurrency int) error {
 	}
 
 	partsToCopy := common.PartsDifference(srcParts, dstParts)
+	partsToCopy = append(partsToCopy, common.FilterPartsForForceCopy(srcParts)...)
```

Comment:
could this function create duplicates? 

 E.g. some parts were added at the previous step to `partsToCopy` and we add it again by filter.

Reply:
Good point, that is possible. I'll update it to only sync those files once